### PR TITLE
Docs: the pipectl download command

### PR DESCRIPTION
**Pull request by sano307**
**What this PR does / why we need it**:

For beginners who are not familiar with the command, I added a download command for pipectl.

**Which issue(s) this PR fixes**:

None

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/user-guide/command-line-tool.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ You can use pipectl to add and sync applications, wait for a deployment status.
 1. Download the appropriate version for your platform from [PipeCD Releases](https://github.com/pipe-cd/pipe/releases).
 
     We recommend using the latest version of pipectl to avoid unforeseen issues.
+    Please set `${VERSION}` to the same format like `v0.9.2`.
+
+    ``` console
+    curl -Lo ./pipctl https://github.com/pipe-cd/pipe/releases/download/${VERSION}/pipectl_${VERSION}_darwin_amd64
```
Comment (nghialv) [inline docs/content/en/docs/user-guide/command-line-tool.md:22]:
```suggestion
    curl -Lo ./pipectl https://github.com/pipe-cd/pipe/releases/download/${VERSION}/pipectl_${VERSION}_darwin_amd64
```

**Comment by nghialv** [comment]
@sano307 Thanks for your pull request. You are absolutely right.
I have just left a comment.

**Comment by sano307** [comment]
@nghialv 
Thank you for the comment. I reflected your code suggestion.

**Comment by nghialv** [comment]
@sano307 Great! Thank you.
/approve

**Pull request merged**